Documentation/Signoffs/Microcontroller.md

```diff
@@ -0,0 +1,46 @@
+# Microcontroller Signoff
+## FUNCTION OF THE SUBSYSTEM
```

Comment:
Essentially this just says the microcontroller will do everything. That is not helpful. Be specific.

Reply:
This was not our intention.  We have specified what functions exactly, it will be doing.